pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -427,11 +427,16 @@ func (n *NetworkPolicyController) getAssociatedGroupsByName(grpName string) []an
 }
 
 // GetGroupMembers returns the current members of a ClusterGroup.
-func (n *NetworkPolicyController) GetGroupMembers(cgName string) (controlplane.GroupMemberSet, error) {
+// If the ClusterGroup is defined by IPBlocks, the returned members will be []controlplane.IPBlock.
+// Otherwise, the returned members will be of type controlplane.GroupMemberSet.
+func (n *NetworkPolicyController) GetGroupMembers(cgName string) (interface{}, error) {
```

Comment:
can we instead return two structs (members, ipb), like how we return (pods, ee) instead of interface?

Reply:
Pushed a new commit that uses this alternative approach